src/rpc/blockchain.cpp

```diff
@@ -2635,7 +2649,9 @@ UniValue CreateUTXOSnapshot(NodeContext& node, CChainState& chainstate, CAutoFil
     result.pushKV("coins_written", stats.coins_count);
     result.pushKV("base_hash", tip->GetBlockHash().ToString());
     result.pushKV("base_height", tip->nHeight);
-
+    result.pushKV("path", fs::PathToString(path));
+    result.pushKV("txoutset_hash", stats.hashSerialized.ToString());
+    result.pushKV("nchaintx", static_cast<int>(tip->nChainTx));
```

Comment:
pretty sure this will overflow soon, if it doesn't already

Reply:
re: https://github.com/bitcoin/bitcoin/pull/23155#discussion_r741921637

> pretty sure this will overflow soon, if it doesn't already

Seems worth addressing. Should this just cast to int64 or is there a more ideal fix?